### ISSUE TITLE
How to get event's drop position?

### ISSUE THREAD

**Issue by damirsel** [CroudSupport/vue-fullcalendar#64]
According to official full calendar docs (https://fullcalendar.io/docs/event_ui/eventDrop/) drop event should hold delta duration object which shows where is dragged event actually dropped. 

In current implementation ("event-drop" event) we only have access to event object once drop is triggered, is there any way to determine event's drop position using current vue-fullcalendar version?

**Comment by BrockReece** [comment]
This lib emits all arguments passed into it from the eventDrop callback, is this broken? 

```js
eventDrop(...args) {
  self.$emit('event-drop', ...args)
},
```

**Comment by damirsel** [comment]
@BrockReece seems to be, since I'm only able to get event object inside "event-drop" event.

**Comment by BrockReece** [comment]
What version of this lib are you using?

**Comment by damirsel** [comment]
0.0.3 for Vue 1.0.28

**Comment by BrockReece** [comment]
Oh right, that is quite an old version, let me rebuild a vue1 project and I will see what I can do

**Comment by BrockReece** [comment]
This should be fixed in v0.0.4

**Comment by damirsel** [comment]
Thanks!
